File: ide/src/main/java/org/dau/ide/project/ProjectManagementPane.java
package org.dau.ide.project;

import javafx.scene.control.Accordion;
import javafx.scene.control.TitledPane;
import org.springframework.beans.factory.annotation.Qualifier;
import org.springframework.stereotype.Component;

@Component
public class ProjectManagementPane extends Accordion {

  public ProjectManagementPane(@Qualifier("management") TitledPane[] panes) {
    super(panes);
  }
}

File: ide/src/main/kotlin/org/dau/ide/project/ProjectManagementPane.kt
package org.dau.ide.project

import javafx.scene.control.Accordion
import javafx.scene.control.TitledPane
import org.springframework.beans.factory.annotation.Qualifier
import org.springframework.stereotype.Component

@Component
class ProjectManagementPane(@Qualifier("management") panes: Array<TitledPane>) : Accordion(*panes) {
  init {
    expandedPane = panes[0]
  }
}
